### PR TITLE
WSTEAMA-268 Reroutes to new homepage

### PR DESCRIPTION
**Pull request by louisearchibald**
Resolves WSTEAMA-268

**Overall change:**
Reroutes to a new homepage via `/tipohome` url

**Code changes:**

- Updates folders and functions for home page
- Updates all home page routes to add `/tipohome`
- Adds appropriate unit tests

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/routes/homePage/getInitialData/index.test.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import fetchMock from 'fetch-mock';
```
Comment (karinathomasbbc) [inline src/app/routes/homePage/getInitialData/index.test.js:1]: We can probably delete most of this 😄 and just check that it returns a status of 200 and an empty `pageData` object

File: src/app/routes/utils/regex/index.test.js [hunk 2]
```diff
@@ -196,6 +221,33 @@ describe('frontPageManifestPath', () => {
...
+  const validRoutes = [
+    '/news/tipohome/manifest.json',
+    '/persian/tipohome/manifest.json',
+  ];
+  shouldMatchValidRoutes(validRoutes, homePageManifestPath);
+
+  const invalidRoutes = [
+    '/foobar/manifest.json',
+    '/foobar/manifest',
+    '/news/trad/sw.js',
```
Comment (karinathomasbbc) [inline src/app/routes/utils/regex/index.test.js]: Let's use some invalid examples, like the `homePageSwPath` ones above

**Review by karinathomasbbc — COMMENTED** [review]
Great stuff - let's simplify the test for the homePage/getInitialData

File: src/app/routes/homePage/getInitialData/index.test.js (hunk 1 above)
Comment (louisearchibald) [reply src/app/routes/homePage/getInitialData/index.test.js:1]: Test added!

File: src/app/routes/utils/regex/index.test.js (hunk 2 above)
Comment (louisearchibald) [reply src/app/routes/utils/regex/index.test.js]: Fixed :)

**Review by pvaliani — APPROVED** [review]

File: src/app/routes/utils/regex/index.test.js [hunk 3]
```diff
@@ -196,6 +221,33 @@ describe('frontPageManifestPath', () => {
...
+  const validRoutes = [
+    '/news/tipohome/manifest.json',
+    '/persian/tipohome/manifest.json',
+  ];
+  shouldMatchValidRoutes(validRoutes, homePageManifestPath);
+
+  const invalidRoutes = [
+    '/news/articles/tipohome/sw.js',
+    '/hausa/tipohome/sw',
+    '/hausa/simp/tipohome/sw.js',
```
Comment (karinathomasbbc) [inline src/app/routes/utils/regex/index.test.js]:
```suggestion
    '/news/articles/tipohome/manifest.json',
    '/hausa/tipohome/manifest',
    '/hausa/simp/tipohome/manifest.json',
```

**Review by karinathomasbbc — APPROVED** [review]
Just one tiny fix to the test, and we're good to go!

File: src/app/routes/utils/regex/index.test.js (hunk 3 above)
Comment (louisearchibald) [reply src/app/routes/utils/regex/index.test.js]: Fixing now 😄

**Pull request merged**